web/src/components/tasks/business-formation/PaymentSection.tsx

```diff
@@ -124,7 +124,7 @@ export const PaymentSection = (props: Props): ReactElement => {
           <Button
             style="secondary"
             onClick={() => {
-              setTab(2);
+              setTab(3);
```

Comment:
Could this be `state.tab + 1`?

Reply:
will update

Reply:
Perhaps an object that maps from sections to tab numbers? Like for example:

```
{
  MEMBER_SECTION: 2
   PAYMENT_SECTION: 3,
}
```

(or whatever the sections/values). That way the code would at least reference the sections and not the hardcoded numbers?

Reply:
updated